sql/sys_vars.cc

```diff
@@ -1678,13 +1678,22 @@ static Sys_var_ulong Sys_expire_logs_days(
        GLOBAL_VAR(expire_logs_days),
        CMD_LINE(REQUIRED_ARG), VALID_RANGE(0, 99), DEFAULT(0), BLOCK_SIZE(1));
 
+static Sys_var_ulonglong Sys_binlog_space_limit(
+       "binlog_space_limit", "Maximum space to use for all binary logs. "
+       "Default is 0, this feature is disabled.",
+       READ_ONLY GLOBAL_VAR(binlog_space_limit), CMD_LINE(REQUIRED_ARG),
+       VALID_RANGE(0, ULONG_MAX), DEFAULT(0), BLOCK_SIZE(1));
```

Comment:
ulonglong var - ULLONG_MAX for the maximum allowed value?

Reply:
This is copied from `relay_log_space_limit`. `ULONG_MAX` depends on CPU architecture:
ULONG_MAX Value (64-bit platforms): 18446744073709551615
ULONG_MAX Value (32-bit platforms): 4294967295